### PR TITLE
RHDM-1159: Exception while coverting spreedsheet decision table to guided decision table

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.redhat.com/browse/RHDM-1159

The XLS attached to the JIRA now converts correctly (and the User can view _source_).

Part of an ensemble:
- https://github.com/kiegroup/drools/pull/2723
- https://github.com/kiegroup/drools-wb/pull/1282

### CODE REVIEW

**Review by Rikkola — APPROVED** [review]

File: drools-workbench-models/drools-workbench-models-commons/src/main/java/org/drools/workbench/models/commons/backend/rule/RuleModelDRLPersistenceImpl.java [hunk 1]
```diff
@@ -2783,7 +2784,16 @@ private static String findOperator(String expr) {
...
     private static boolean isNotMethodName(final String expression,
+                                           final String potentialOperator,
                                            final int operatorPosition) {
+        if (Objects.equals(potentialOperator, Operator.EQUAL.getOperatorString()) ||
+                Objects.equals(potentialOperator, Operator.NOT_EQUAL.getOperatorString()) ||
+                Objects.equals(potentialOperator, Operator.LESS.getOperatorString()) ||
+                Objects.equals(potentialOperator, Operator.LESS_OR_EQUAL.getOperatorString()) ||
+                Objects.equals(potentialOperator, Operator.GREATER.getOperatorString()) ||
+                Objects.equals(potentialOperator, Operator.GREATER_OR_EQUAL.getOperatorString())) {
+            return true;
```
Comment (jomarko) [inline drools-workbench-models/drools-workbench-models-commons/src/main/java/org/drools/workbench/models/commons/backend/rule/RuleModelDRLPersistenceImpl.java:2795]: Shouln't we use `org.drools.core.base.evaluators.Operator.getAllOperators()` ?

**Review by jomarko — COMMENTED** [review]
### Code review

**Comment by jomarko** [comment]
Manaul review postponed until comment from code review responded.

File: drools-workbench-models/drools-workbench-models-commons/src/main/java/org/drools/workbench/models/commons/backend/rule/RuleModelDRLPersistenceImpl.java (hunk 1 above)
Comment (manstis) [reply drools-workbench-models/drools-workbench-models-commons/src/main/java/org/drools/workbench/models/commons/backend/rule/RuleModelDRLPersistenceImpl.java:2795]: No, this addition came about due to the XLS attached to the JIRA containing a CONDITION definition `traceId==("$param")` where there are no spaces between the operator and the value so [this](https://github.com/kiegroup/drools/blob/9ea375f2be1d54d0598f35ad571681c798fec1f9/drools-workbench-models/drools-workbench-models-commons/src/main/java/org/drools/workbench/models/commons/backend/rule/RuleModelDRLPersistenceImpl.java#L2797) check failed and the parsing of the XLS lead to incorrect results. 

I only check for `==`, `!=`, `<`, `<=`, `>`, `>=` as these cannot be the prefix to a method name whereas, for example, `matches` could be in the method name `matchesCheese()`.

**Review by jomarko — APPROVED** [review]
1/2

**Comment by manstis** [comment]
@mareknovotny Sorry to ask.. how do I trigger a "downstream compile" (for the failed check above)?

**Comment by jomarko** [comment]
jenkins execute compile downstream build

**Pull request merged**